eightpoints/guzzle-bundle/7.0/config/packages/dev/eight_points_guzzle.yaml

```diff
@@ -0,0 +1,2 @@
+eight_points_guzzle:
+    logging: true
```

Comment:
The default value for this parameter is KERNEL_DEBUG, right?

Reply:
yep, `%kernel.debug%`